From a7377e0cca4f63ca4ce56bef87ac1f82dd219970 Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Fri, 28 Mar 2014 12:03:46 +0100
Subject: [PATCH 09/48] Autostart Baloo in GNOME, Unity and XFCE

Baloo should always be running otherwise it will not pick up file
changes, specially moves.
---
 src/file/baloo_file.desktop | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/file/baloo_file.desktop b/src/file/baloo_file.desktop
index 7fb8a72..c02bd58 100644
--- a/src/file/baloo_file.desktop
+++ b/src/file/baloo_file.desktop
@@ -3,8 +3,11 @@ Type=Service
 Exec=baloo_file
 X-KDE-StartupNotify=false
 X-KDE-autostart-condition=baloofilerc:Basic Settings:Enabled:true
-OnlyShowIn=KDE;
 X-KDE-autostart-phase=0
+X-GNOME-Autostart-enabled=true
+OnlyShowIn=KDE;GNOME;Unity;XFCE
+NoDisplay=true
+
 Name=Baloo File Daemon
 Name[bs]=Baloo datoteƒçni demon
 Name[ca]=Dimoni de fitxers Baloo
-- 
1.9.1

